docs/README.md

```diff
@@ -0,0 +1,8 @@
+# single-spa documentation
```

Comment:
This Readme is shown by default when you go to /docs, which is linked to from the main readme on the home page.
  
Beforehand there was nothing showing, so hopefully this is a little more convenient for people